cmd/antrea-agent/agent.go

```diff
@@ -844,6 +847,21 @@ func run(o *Options) error {
 		go mcStrechedNetworkPolicyController.Run(stopCh)
 	}
 
+	klog.InfoS("Waiting for flow restoration to complete")
```

Comment:
@tnqn I wasn't sure what would be the best place for this code. I put it here because I think by that point all the controllers that need to install flows have been instantiated, initialized and "started" (even though at the moment, `flowRestoreCompleteWait` only waits for the NetworkPolicyController + NodeRouteController + CNIServer).

Reply:
It makes sense to me. I tried to delay FlowRestoreComplete but was unsure whether moving ConnectUplinkToOVSBridge has any impact. cc @gran-vmv

Reply:
We can run flexible-ipam-e2e multiple times to check if this change can work.